### PR TITLE
Adjusting quantity on product page if empty or under min qty

### LINKED ISSUE #10125: Bad behavior with qty and add to cart
> **Reminder**
> 
> DO NOT create security issues here, but contact security@prestashop.com instead!

**Describe the bug**

**To Reproduce**
1. Go to any page of a product
2. Inset empty value in qty field.
3. Click on 'Add to cart'
4. There are not added any product to the cart. In case the theme use a spinning icon while add products to cart, this icon will spin until you reload the page.
The behavior is wrong. In case of empty qty field, when click on add to cart, the qty field should increase to 1 and add 1 to cart.

**Screenshots**
screenshot
When click on add to cart, the value should change from empty to 1 and add 1 pcs to cart.

**Additionnal information**
PrestaShop version: 1.7.3
PHP version: 7.1.x

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | If qty on product page was empty, you had an infinite loop
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #10125 
| How to test?  | Go on product page, kick off value of qty click out and it should change to min qty product settings, also try a value under the min product settings it should change to min

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16965)

### CODE REVIEW

File: themes/classic/_dev/js/product.js [hunk 1]
```diff
@@ -106,12 +106,24 @@ $(document).ready(function () {
       max: 1000000
     });
 
+
+    let $inputQty = $('#quantity_wanted');
```
Comment (arouiadib) [inline themes/classic/_dev/js/product.js]: Hi Valentin, thank you for this PR which improves clearly UX. I have tested your branch and it is working nicely, I just have one question: why not to use the constant `$quantityInput` already used in line 97?

**Review by matks — DISMISSED** [review]

File: themes/classic/_dev/js/product.js (hunk 1 above)
Comment (NeOMakinG) [reply themes/classic/_dev/js/product.js]: Hey, shame on me, correcting that right now :D

Comment (NeOMakinG) [reply themes/classic/_dev/js/product.js]: Done. Thanks for your review, appreciate it

**Review by matks — DISMISSED** [review]

File: themes/classic/_dev/js/product.js (hunk 1 above)
Comment (arouiadib) [reply themes/classic/_dev/js/product.js]: It happens to all of us :).  Thank you Valentin

File: themes/classic/_dev/js/product.js [hunk 2]
```diff
@@ -106,12 +106,21 @@ $(document).ready(function () {
       max: 1000000
     });
 
+    $quantityInput.focusout((e) => {
```
Comment (PierreRambaud) [inline themes/classic/_dev/js/product.js]: e is not used 
```suggestion
    $quantityInput.focusout(() => {
```

**Review by PierreRambaud — APPROVED** [review]

**Comment by sarahdib** [comment]
@NeOMakinG 
If I set up 0 in qty field -> the field change it to 1 -> OK
If I kick off  the qty -> nothing happen -> Not OK

**Comment by NeOMakinG** [comment]
@sarahdib you need to click out of the case to see the input changing, as specified in the how to test part, we cant validate on type, did as its specified by Pablo in the issue !

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**